tools/pipelines/templates/1ES/include-policy-check.yml

```diff
@@ -0,0 +1,69 @@
+# Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+# Licensed under the MIT License.
+
+# include-policy-check
+#
+# This template can be included in pipelines that run check scripts as part of their pipeline. The check scripts are
+# executed serially in a separate stage from the main build.
```

Comment:
Nit (for the future): Could we expand the documentation to explain what we mean by "checks"?

Reply:
Actually, after reading through the code, it "checks" seems pretty open-ended. It seems like the purpose of this template is really just to run a series of npm scripts after installing node (and then verifying that nothing changed, git-wise). This feels much more general than "policy-check". I wonder if we have other templates that have essentially the same structure that we could deduplicate.

Reply:
These are future considerations though :)